.github/workflows/jest-coverage-changes.yml

```diff
@@ -0,0 +1,26 @@
+name: Jest Coverage - changed files
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - release/v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

Comment:
Can use this instead :- 
```      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: lts/*
      - name: Install dependencies
        run: npm install -g yarn && yarn
```
this takes care of installing yarn as well if not present

Reply:
Done